tests/unit/droppable/droppable_events.js

```diff
@@ -60,4 +58,43 @@ test("drop", function() {
 });
 */
 
+test( "drop event callback order", function() {
+	expect( 4 );
+
+	$( "<div id='droppable3'></div>" ).appendTo( "body" );
+	$( "<div id='droppable4'></div>" ).appendTo( $( "#droppable3" ) );
+	$( "<div id='draggable2'></div>" ).appendTo( "body" );
+	var droppable3 = $( "#droppable3" ),
+		droppable4 = $( "#droppable4" ),
+		draggable2 = $( "#draggable2" );
+
+	droppable3.css({ position: 'absolute', width: 100, height: 100, left: 0, top: 0 });
+	droppable4.css({ position: 'absolute', width: 50, height: 50, left: 0, top: 0 });
```

Comment:
Always use double quotes for strings. Might as well just chain these into the creation.